backend/index.js

```diff
@@ -0,0 +1,82 @@
+const Recipe = require("./models/recipeSchema")
+
+const express = require('express')
+const app = express()
+app.use(express.json())
+const mongoose = require('mongoose')
+const connection_url = "mongodb+srv://newUser1:notmypassword@uswest.e7u3b.mongodb.net/RecipesDB?retryWrites=true&w=majority"
+
+
+mongoose.connect(connection_url)
+    .then(() => console.log('Successfully connected'))
+    .catch((error) => console.error(`Could not connect due to ${error}`))
+
+app.get('/', (req, res) => {
+  res.send('Hello world!')
+})
+
+// GET - /api/recipe
+app.get("/api/recipe", async (req, res) => {
+    const recipes = await Recipe.find({})
+    res.send(recipes)
+})
+
+// GET - /api/recipe/:recipeName
```

Comment:
Good use of comments. It really does make code a lot more organized and readable. Also, this is totally not necessary, but it is also helpful to write a short description of what the route is doing!